### ISSUE TITLE
Disappearing funds in Copay (receiver and sender)

### ISSUE THREAD

**Issue by bearwhalebandit** [bitpay/copay#7418]
Hi there,

Experiencing a very major issue and this needs to be brought to the community and Copay's attention. I've seen there are similar support tickets for the same reason. I have yet to resolve my issue.

On December 9th, 8:45am PST, a payment of 0.043239 BTC was sent to my wallet address 1LqRQgKFDKfm6za8q6esdU2yFYe1zngqJc with a fee of approximately 100 sat/byte.

It showed up in the wallet, and was deducted from sender's balance (We are both using Copay for Android).

From here, it has been going back and forth between being added and removed from my balance. The balance continues to fluctuate everytime I refresh. It shows as receiving 0.0432239, then it disappears from my transaction history. The same thing is happening on the sender's Copay. Where it shows he sent it, then didn't. In either case, the funds are not spendable for me, nor are they resendable for him.

When viewing on ANY block explorer, it doesn't show up. This includes insight, Blockchain.info. Block Cypher etc. *** COPAY SUPPORT: THIS IS NOT RELATED TO INSIGHT NOT WORKING ***  The transaction was never submitted on the Blockchain, yet the funds are missing on both ends. This leaves me to say with 100% certainty it is stuck bouncing around in Copays servers. Please be advised, I am not a beginner when it comes to BTC. I own a business that uses crypto daily and makes several dozens of BTC payments daily. I  have done various intermediate to advanced troubleshooting and still my funds are missing, in limbo.  These includes restoring on other wallets, doing standard blockchain analysis, deriving the Extended Private Key, and everything else possible to determine this is specific to Copay. 

I can show videos of the balance coming and disappearing, on both devices. I have screenshots of this as well. I have the wallet that was meant to receive the payment above (1LqRQgKFDKfm6za8q6esdU2yFYe1zngqJc).

Transaction ID is 83309d5858cf226c3c0edd360c36a2c0c059db3da3a3582f47df6cd468bf2f80 however this doesn't apply as it is not registered on the Blockchain and cannot be viewed.

A resolution to this matter would be appreciated.

**Comment by omietrice** [comment]
I have brought this issue up. the community nor Reddit cares and they have seemed to mark my post as spam. View the post here.. I have lost 2 seperate transactions already totalling over $3000.


https://www.reddit.com/r/Bitcoin/comments/7j31d5/copay_stealing_funds_upvote_this_so_no_one_is/

**Comment by bearwhalebandit** [comment]
Javier, your assistance would be appreciated

**Comment by bearwhalebandit** [comment]
There have been numerous people with this same issue since I posted this. We need responses

**Comment by telexqwerty** [comment]
Hi, now I see your thread. This is exactly my case too. I sent to my Bitpay wallet.
Then in the Copay it's "sending" back and forth and in Bitpay it's "receiving" back and forth.
Is this the bug of both apps?

We are waiting for response and resolution.Copay/Bitpay please help!!!

**Comment by inlerfect** [comment]
We are currently experience synchronization issues between our servers and the blockchain because of low tx fees. We are working hard to resolve them, but it can take a few weeks before your transactions to show up. If you need your wallet in a synchronized state immediately **do the following**:

1. Send your balance to 1Copay77sYUEyrD1tLjuRgzPe3pGLZHR8F
2. Use a higher than normal fee for this transaction.
3. Our systems will give preference to this transaction and returns the coins to your wallet. This might take up to 30 minutes.

**Comment by Schknheit** [comment]
Scam^ The profile was created today and has nothing to do with Bitpay/Copay

**Comment by bearwhalebandit** [comment]
Thanks, I was wondering the same thing ... Seems suspicious.

Please check out this Reddit thread posted by another user with the same issue.

https://www.reddit.com/r/Bitcoin/comments/7keqmn/community_we_need_to_upvote_this_post_so_no_one/

**Comment by QuestBit** [comment]
Having the same isssue. Didn’t receive 2 Btc that been sent out to the receiver.

**Comment by Gamboster** [comment]
Please, do not send your funds as indicated by @inlerfect . He is a scammer taking advantage of the situation. Copay developers will never ask you to send your funds in that way.

**Comment by carlosgfranco** [comment]
Same issue here: https://github.com/bitpay/copay/issues/7807 Any solutions???

**Comment by matiu** [comment]
Hello, I am checking this issue again, sorry the delay.

So, check the address https://blockchain.info/address/1LqRQgKFDKfm6za8q6esdU2yFYe1zngqJc,
there is no incomming TX to it. 

Please note that is the TX was sent to you with a fee too low, it can be dropped off the mempool, and after that the funds should appear to the sender.

If the sender is using a Copay wallet, please let us know to indicate the walletID or any wallet address.

**Comment by rosdi** [comment]
Just would like to chip in... I have successfully transferred out my bitcoins out of BitPay wallet (which I believe is almost identical to Copay) to a hardware wallet address. I did this twice! I used 50 sat/b and then a few days later 30 sat/b... all went through without hiccup and was confirmed in under an hour.

First transaction was from a 'normal' address..., the second transaction was from multi-sig address. Both got confirmed pretty quickly considering the low fee I was using.

I know my answer does not help this thread in any way... but hopefully I can help clear the FUD surrounding Copay & BitPay wallet... cheers.
